data_structures/segment_tree.c

```diff
@@ -0,0 +1,162 @@
+#include <string.h>
```

Comment:
```suggestion
/**
 * @file 
 * @brief Add one line description here
 * @details 
 * This is a multi line
 * description containing links, references,
 * math equations, etc
 * @author [Name](https://github.com/handle)
 */

#include <string.h>
```
This initial comment block is required.

Reply:
I have fixed that also

Reply:
@kvedala